process/sig-lifecycle.md

```diff
@@ -1,13 +1,30 @@
 # Special Interest Group (SIG) Life Cycle
 
-Special Interest Groups are under the direct governance of a Working Group (WG) which can create and close them as necessary and appropriate. SIGs are expected to have a very specific goals and objectives at the time of their creation which may or may not define if they are terminable and is entirely dependent on the nature of the SIG.
+Special Interest Groups are under the direct governance of their reporting Working Group (WG) and are bound to achieving a very specific goal. These groups may be terminated upon completion of their designating tasking, continued for larger and ongoing efforts, or otherwise subject to the governance, structure, and termination policies of the WG they are under. The creation of a SIG must dictate the focus, intent, goals, and deliverables as appropriate.
 
-The lifecycle of SIGs is therefore minimal and as follows.
+It is expected that the primary output of a SIG is not software. If the primary output is software, the work should be organized as a [project](./project-lifecycle.md).
 
-## Active
+SIG process should be minimal with as much as possible delegated to the WG. However we do need minimal process to at least ensure we have an accurate list of all the active SIGs in the OpenSSF. This document uses "must" to describe what items are required and "should" to suggest items that should be strongly considered (but not required).
 
-* The SIG is actively pursuing its goal.
+## To become `Tentative`:
```

Comment:
I landed on `Tentative` as a term that includes both the connotation that it's in an early phase of development, without trying to imply that it has endorsement (like `Proposed` might).

Reply:
How about "Forming"?

Reply:
I think `Forming` still could imply some level of endorsement - I'm going to stick with `Tentative`.